content-learn/mastering-meshery-istio/performance-of-service-mesh/istio/circuit-breaking.mdx

```diff
@@ -0,0 +1,127 @@
+---
+docType: "Chapter"
+chapterTitle: "Circuit Breaking"
+description: "Meshery is the service mesh managemen plane which offers lifecycle, configuration, and performance management of service meshes and their workloads."
+videos: 4
+lectures: 12
+---
+
+import { ChapterStyle } from "../../../../src/components/Learn-Components/Chapters-Style/chapters.style.js";
+import MesheryInitialLoadTest from "../../../../src/assets/images/learn-layer5/istio/meshery_initial_load_test.png";
+import MesheryCbLoadTest from "../../../../src/assets/images/learn-layer5/istio/meshery_cb_load_test.png";
+
+<ChapterStyle>
+
+In this chapter we will configure circuit breaking using Istio. Circuit breaking allows developers to write applications that limit the impact of failures, latency spikes, and other undesirable effects of network peculiarities. This task will show how to configure circuit breaking for connections, requests, and outlier detection.
+
+<h2 class="chapter-sub-heading">Preparing for circuit breaking</h2>
+<br />
+
+Before we can configure circuit breaking, please try to access the `product page` app from within `Meshery` to ensure all the calls are making it through **without** errors as we did in [Observability chapter](/learn-ng/mastering-meshery-istio/performance-of-service-mesh/istio/observability) (<a href={MesheryInitialLoadTest}>see screenshot</a>).
```

Comment:
The `see screenshot` link is missing the image.

Reply:
Its imported and referenced inside href 🤔